### PR TITLE
Update guest_version.sh

### PR DESCRIPTION
**Pull request by Maunty**
Script updated in order to provide branch name along with commit id

### CODE REVIEW

**Comment by Maunty** [comment]
tested with https://ci.arenadata.io/job/gpdb/job/adb-6.x/23/console.

File: gues_version.sh [hunk 1]
```diff
@@ -14,7 +14,36 @@
...
+  origin/pr/* )
+    # we dont build adcm pr, only branches, just in case
+    current_branch="_$(echo "$name_rev" | cut -f 2,3 -d '/' | tr '/' '-')"
+    ;;
+  *)
+    current_branch="_$name_rev"
+    ;;
+esac
+
+printf '{\n\t"version": "%s",\n\t"commit_id": "%s"\n}\n' "${current_date}" "${current_hash}${current_branch}" > config.json
```
Comment (shvepsy) [inline gues_version.sh:46]:
```suggestion
printf '{\n\t"version": "%s",\n\t"commit_id": "%s",\n\t"branch": "%s"\n}\n' "${current_date}" "${current_hash}" "${current_branch}" > config.json
```

Comment (shvepsy) [reply gues_version.sh:46]: Can drop it if config.json isn't support `branch` field

**Review by shvepsy — APPROVED** [review]

File: gues_version.sh (hunk 1 above)
Comment (Maunty) [reply gues_version.sh:46]: I think that we need @acmnu decision how is better to do. Currently i don't know what is frontend algorithm to gather adcm version from config,  so i decided to hack commit_id value.

**Pull request closed without merging**